lego/apps/users/tests/test_users_api.py

```diff
@@ -359,6 +359,16 @@ def test_update_not_failing_when_not_student(self):
         user = self.all_users.exclude(student_username__isnull=False).first()
         self.successful_update(user, user)
 
+    def test_update_to_remove_profile_picture(self):
+        """Try to remove profile picture by updating user with profilePIcture equals null"""
+        user = self.all_users.exclude(student_username__isnull=False).first()
```

Comment:
Should also check that user already has a picture, so that the change actually has an effect